### PR TITLE
render change

### LINKED ISSUE #34: the page / does not point to welcome
change home Controller

### PR DESCRIPTION
**Pull request by bouboumael**

### CODE REVIEW

**Review by Loic-Code — DISMISSED** [review]

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -21,6 +21,6 @@ class HomeController extends AbstractController
      */
     public function index()
     {
-        return $this->twig->render('Home/index.html.twig');
+        return $this->twig->render('User/welcome.html.twig');
```
Comment (LucyMerlier) [inline src/Controller/HomeController.php]: it's better to put the templates that a controller renders in a folder with the name of the controller (HomeController => renders templates in "Home" folder)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

File: src/Controller/HomeController.php (hunk 1 above)
Comment (bouboumael) [reply src/Controller/HomeController.php]: ok correction done

**Review by Loic-Code — DISMISSED** [review]

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "twig-language-2.indentStyle": "space",
+    "twig-language.indentStyle": "space"
+}
```
Comment (LucyMerlier) [inline .vscode/settings.json]: your .vscode file should not be versionned

**Review by LucyMerlier — DISMISSED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (bouboumael) [reply .vscode/settings.json]: ok deleted

**Review by Loic-Code — DISMISSED** [review]

**Review by LucyMerlier — DISMISSED** [review]

**Review by Loic-Code — APPROVED** [review]

**Review by ChristianOlmedo — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**